source/dubregistry/dbcontroller.d

```diff
@@ -344,9 +353,7 @@ class DbController {
 			pkg.textScore = (pkg.textScore - minMaxTS[0]) * scale + DbPackageStats.minScore;
 
 		// sort found packages by weighted textScore and package score
-		return pkgs
-			.sort!((a, b) => a.stats.score + 2 * a.textScore > b.stats.score + 2 * b.textScore)
-			.release;
+		return pkgs.sort!((a, b) => a.stats.score + 2 * a.textScore > b.stats.score + 2 * b.textScore).release;
```

Comment:
there is no textScore anymore because you aren't doing a `$text` query that fills it. Pretty sure this is a no-op right now